Adding SRW results (#21)
dtcenter/ufs-case-studies: docs/source/2020CAD.rst

   
      crontab 2020CAD.crontab  

  .. group-tab:: SRW.v1.0

>     The case runs are initialized at 12z Feb 03, 2020 with 90 hours forecasting.The app uses ``config.sh`` to define the runtime settings. The settings that need to be modified to set up the first cycle, last cycle, forecast length and cycle hour are listed below. 

Reviewer: in all places, I recommend to replace `with XX hours forecasting` with `with a forecast length of XX hours`.

    * GFS.v16.0.10 shows a tendency of the positively tilted trough at 500 hPa, which is consistent with GFS_ANL.

  .. group-tab:: SRW.v1.0

    .. figure:: images/2020CAD/MSLP_SRW_v1.0_2020CAD_trim.png
      :width: 1200
      :align: center

      Mean sea level pressure (hPa)

>     * SRW_RRFSv1alpha more correctly forecasts the surface low than SRW_GFSv15p2.

Reviewer: SRW_RRFSv1alpha forecasts the surface low more accurately than SRW_GFSv15p2.

Reviewer: ```suggestion
    * SRW_RRFSv1alpha forecasts the surface low more accurately than SRW_GFSv15p2.
```

    * Stronger surface winds are simulated by GFS.v16.0.10.

  .. group-tab:: SRW.v1.0

    .. figure:: images/2020CAD/2mT_SRW_v1.0_RAP_2020CAD_RAP_trim.png
      :width: 1200
      :align: center

      2-m temperature (F) 

>     * Cold air was dammed up along the Appalachian mountains in RAP_ANL.

Reviewer: I don't know if you can say "was dammed up" @ligiabernardet

Reviewer: ```suggestion
    * Cold air was blocked by the Appalachian mountains in RAP_ANL.
```

>     * Both SRW_RRFSv1alpha and SRW_GFSv15p2 generates warm bias along the east side of Appalachian.

Reviewer: generate (remove the "s")

Reviewer: ```suggestion
    * Both SRW_RRFSv1alpha and SRW_GFSv15p2 generate warm bias along the east side of Appalachian.
```
